docs/_pages/v2/guide-authentication.md

```diff
@@ -81,6 +81,29 @@ curl --user ${NETFOUNDRY_CLIENT_ID}:${NETFOUNDRY_PASSWORD} \
     --data 'grant_type=client_credentials&scope=https%3A%2F%2Fgateway.production.netfoundry.io%2F%2Fignore-scope'
 ```
 
+### NodeJS
+
+```
```

Comment:
Please add `javascript` to the end of this line to enable syntax highlighting for the code block